test/e2e/upgrade/common.go

```diff
@@ -65,7 +65,7 @@ func subscription(olm tutils.OLMEnv) *coreos.Subscription {
 func createAndPopulateVolatileCache(cacheName string, numEntries int, client tutils.HTTPClient) {
 	c := tutils.NewCacheHelper(cacheName, client)
 	if !c.Exists() {
-		c.Create(`{"distributed-cache":{"mode":"SYNC"}}`, mime.ApplicationJson)
+		c.Create(`{"distributed-cache":{"mode":"SYNC", \"encoding\": {\"media-type\": \"application/json\"}}}`, mime.ApplicationJson)
```

Comment:
No need for the backslashes in this file as we're using a ` string in both cases.

We're getting the error "unexpected HTTP status code (400): unexpected error creating cache, response: Invalid JSON at [1,72]" without testing locally, I'm not sure if that's to do with ^ formatting, or if the 14.0.x images don't like the json.